package.json

```diff
@@ -31,8 +31,8 @@
   },
   "homepage": "https://github.com/mozilla/webmaker-android",
   "devDependencies": {
-    "autoless": "^0.1.7",
     "babel": "^5.3.3",
+    "autoless": "git://github.com/k88hudson/autoless.git#notify",
```

Comment:
Is the thing thing is forked for also filed as PR upstream so we can go back to the next version of autoless in the future? (also a small nit: preserve alphabetical ordering?)